scripts/run_unitests.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 set -x
 [ -n "$1" ] && coverage="-v $1:/driver/coverage:z"
-docker build -f Dockerfile-csi-controller.test -t csi-controller-tests . && \
+docker build -f Dockerfile-controllers.test -t csi-controller-tests . && \
```

Comment:
do we have a ticket to add local tests for the host definition feature?

Reply:
CSI-4344